### PR TITLE
Add some new feedback types that send to different emails

### PR DESCRIPTION
**Pull request by AWare**
## Why are you doing this?
So that membership feedback is no longer flooded.

### CODE REVIEW

File: frontend/app/forms/MemberForm.scala [hunk 1]
```diff
@@ -108,7 +108,7 @@ object MemberForm {
     lazy val zuoraAccountAddress = billingAddress.getOrElse(deliveryAddress)
   }
 
-  case class FeedbackForm(category: String, page: String, feedback: String, name: String, email: String)
+  case class FeedbackForm(category: FeedbackType, page: String, feedback: String, name: String, email: String)
```
Comment (rtyley) [inline frontend/app/forms/MemberForm.scala]: I realise this was already here, but I it looks to me like `FeedbackForm` doesn't belong in `frontend/app/forms/MemberForm.scala` - should go in it's own file, especially now it's expanding with the `feedbackTypeFormatter`

File: frontend/app/services/EmailService.scala [hunk 2]
```diff
@@ -10,16 +10,16 @@ import com.gu.identity.play.IdMinimalUser
...
 import forms.MemberForm.FeedbackForm
 
 import scala.util.{Failure, Success, Try}
 
 trait EmailService extends LazyLogging {
-  val feedbackAddress: String
 
   val client = {
-    val c = new AmazonSimpleEmailServiceClient()
+    val c = new AmazonSimpleEmailServiceClient(CredentialsProvider)
```
Comment (rtyley) [inline frontend/app/services/EmailService.scala]: :+1:

File: frontend/app/model/FeedbackType.scala [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  override val description: String = "an accessibility issue"
+}
+
+case object MembershipFeedback extends FeedbackType {
+  override val description: String = "feedback on the Membership scheme"
+  override val email: String = membershipFeedback
+}
+
+case object GuardianLive extends FeedbackType {
+  override val description: String = "feedback on the Membership scheme"
```
Comment (rtyley) [inline frontend/app/model/FeedbackType.scala]: `"feedback on the Membership scheme"` is probably a copy-paste from `MembershipFeedback`, should be different for `GuardianLive`?

File: frontend/app/forms/MemberForm.scala [hunk 4]
```diff
@@ -204,7 +219,7 @@ object MemberForm {
   )(PaymentForm.apply)(PaymentForm.unapply)
 
   val feedbackMapping: Mapping[FeedbackForm] =   mapping(
```
Comment (rtyley) [inline frontend/app/forms/MemberForm.scala]: This variable (`feedbackMapping`) is unused, you can remove it, and save 5 lines!

File: frontend/app/forms/FeedbackForm.scala [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+import play.api.data.Forms._
+import play.api.data.format.Formatter
+import play.api.data.{Form, FormError, Mapping}
+import model.FeedbackType
+
+/**
+  * Created by alex_ware on 20/01/2017.
+  */
+object FeedbackForm {
+  case class FeedbackForm(category: FeedbackType, page: String, feedback: String, name: String, email: String)
```
Comment (rtyley) [inline frontend/app/forms/FeedbackForm.scala]: This should be a companion case class to  the `FeedbackForm` object! ie bring this case class up one level

File: frontend/app/views/fragments/form/feedback.scala.html [hunk 6]
```diff
@@ -61,7 +61,9 @@ <h2 class="fieldset__headline">About you</h2>
                    class="input-text"
                    type="text"
                    required
-                   aria-required="true"/>
+                   aria-required="true"
+                   value="@name.getOrElse("")"
```
Comment (rtyley) [inline frontend/app/views/fragments/form/feedback.scala.html]: Alternative for `name.getOrElse("")` is `name.mkString`, but your taste may differ!

File: frontend/app/forms/FeedbackForm.scala (hunk 5 above)
Comment (AWare) [inline frontend/app/forms/FeedbackForm.scala]: oops, fixed

File: frontend/app/model/FeedbackType.scala [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+trait FeedbackType {
+  val description: String
+  val email: String = membershipSupport
+  def slug: String = description.filter(_.isLetterOrDigit).toLowerCase
+}
+
+object FeedbackType {
+  val all: Seq[FeedbackType] = List(CustomerService, AccessibilityIssue, MembershipFeedback, GuardianLive, Journalism)
+  def fromSlug(slug: String) = all.find(_.slug == slug)
+}
```
Comment (rtyley) [inline frontend/app/model/FeedbackType.scala]: Seeing all this makes me think we should try https://github.com/lloydmeta/enumeratum - maybe in a different PR

**Pull request merged**